File: JavaScript/baekjoon/bronze/baek_2588.js
// 백준 2588번
// 곱셈
// 입출력, 사칙연산, 수학

const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().split("\n");

const a = parseInt(input[0].trim());
const b = parseInt(input[1].trim());
const b1 = b % 10;
const b2 = Math.floor((b % 100) / 10);
const b3 = Math.floor(b / 100);

console.log(a * b1);
console.log(a * b2);
console.log(a * b3);
console.log(a * b);

File: Python/baekjoon/bronze/baek_2588.py
## 백준 2588번
## 곱셈
## 입출력, 사칙연산, 수학

A = int(input())
B = input()

for i in range(2, -1, -1):
    print(A*int(B[i]))
print(A*int(B))